components/calendar/src/cal/chinese.rs

```diff
@@ -141,11 +146,17 @@ pub trait Rules: Clone + core::fmt::Debug + crate::cal::scaffold::UnstableSealed
 
 /// The [`Rules`] used in China.
 ///
-/// This type agrees with the official data published by the
+/// # Precise definition and limits
+///
+/// This calendar is intended to represent the traditional Chinese lunar calendar as used
+/// officially in the People's Republic of China as of 2025. This takes a best-effort approach
```

Comment:
let's try to be generic over "China", this is also used in Taiwan

```suggestion
/// officially in China as of 2025. This takes a best-effort approach
```

Reply:
I think "China" is better because the calendar is used throughout the region. If different sub-regions adopt different rules, then we might want to get more specific with the naming. And in that case, I might say "in Beijing" rather than "in China".